<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(198, 165, 126);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" alignment="Qt::AlignVCenter">
       <widget class="QPushButton" name="searchButton">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;
background-color: rgb(178, 145, 103);</string>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" alignment="Qt::AlignVCenter">
       <widget class="QPushButton" name="resetButton">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;
background-color: rgb(178, 145, 103);</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="searchLine">
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="addRow">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;
background-color: rgb(178, 145, 103);</string>
        </property>
        <property name="text">
         <string>Add title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deletRow">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;
background-color: rgb(178, 145, 103);</string>
        </property>
        <property name="text">
         <string>Delete title</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(178, 145, 103);</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="5" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="descriptionLabel">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;</string>
          </property>
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="descriptionLine"/>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="ratingLine">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="6" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="languageLabel">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;</string>
          </property>
          <property name="text">
           <string>Language</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="bookformatLabel">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;</string>
          </property>
          <property name="text">
           <string>Book Format</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLineEdit" name="pagesLine"/>
        </item>
        <item row="1" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="authorLabel">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;</string>
          </property>
          <property name="text">
           <string>Author</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="genresLine"/>
        </item>
        <item row="7" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="genresLabel">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;</string>
          </property>
          <property name="text">
           <string>Genres</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="ratingLabel">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;</string>
          </property>
          <property name="text">
           <string>Rating</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="titleLine">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="9" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="pagesLabel">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;</string>
          </property>
          <property name="text">
           <string>Pages</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="languageLine"/>
        </item>
        <item row="0" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="titleLabel">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Academy Engraved LET&quot;;</string>
          </property>
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLineEdit" name="bookformatLine"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="authorLine">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="ProjectName">
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Academy Engraved LET&quot;;</string>
      </property>
      <property name="text">
       <string>What to read</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
